config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/stakater/workshop-operator
-  newTag: ocp4.6
+  newName: controller
```

Comment:
@MaazMS why name it controller? i think it should be
stakater/workshop-operator

Reply:
It auto generated . I am fixing it .